lib/api/CInferenceModelDefinition.cc

```diff
@@ -491,10 +719,45 @@ const CFrequencyEncoding::TStringDoubleUMap& CFrequencyEncoding::frequencyMap()
     return m_FrequencyMap;
 }
 
+CFrequencyEncoding::CSizeInfo::CSizeInfo(const CFrequencyEncoding& encoding)
+    : CSizeInfo(&encoding) {
+}
+
+CFrequencyEncoding::CSizeInfo::CSizeInfo(const CFrequencyEncoding* encoding)
+    : CEncoding::CSizeInfo::CSizeInfo(encoding), m_Encoding{*encoding} {
+}
```

Comment:
As above. In fact, I have the same comment for all these concrete encoding class objects.